stable/unbound/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Unbound is a fast caching DNS resolver
 home: https://www.unbound.net/
 name: unbound
-version: 0.1.2
+version: 0.1.3
```

since this chart is in stable lets bump that to `1.0.0` https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#versioning

```suggestion
version: 1.0.0
```

---

@NaPs please let me know when you have time to make this change or i can make it as well